### ISSUE TITLE
Solution for using fastboot

### ISSUE THREAD

**Issue by claudio147** [chrislopresto/ember-freestyle#92]
It would be fine if there would be a solution for using ember-freestyle with fastboot. At the moment there are a few warnings like:

> Query params are not included in href attributes when using link-to with ember-cli-staticboot. If you require query params this suggests your app is dynamic and not best suited for ember-cli staticboot.

It's because of the links in the navigation.

**Comment by jonasbuechel** [comment]
Is there any workaround to disable the console output?

**Comment by simonihmig** [comment]
It seems this issue is related to `ember-cli-staticboot`, not FastBoot in general, right?

**Comment by chrislopresto** [comment]
I haven't played around with FastBoot at all yet, but I'd love to pair with someone who has a FastBoot project.

**Comment by simonihmig** [comment]
I have actually played around with FastBoot quite a lot recently. I don't see ember-freestyle currently doing anything that should not work in FastBoot. In fact our app is using both, and I just checked and can confirm, that our styleguide renders just fine on the server!

As I said above, the problem reported here seems to be related to ember-cli-staticboot, not FastBot itself.

I have made an addon to easily test addons for FastBoot compatibilty: https://github.com/kaliber5/ember-fastboot-addon-tests. I could add some tests to really make sure this all works. But maybe after things like #43 have landed, to not have to rewrite all tests again!?

**Comment by bertdeblock** [comment]
:wave: Going through some old issues here.

I tested the current version of `ember-freestyle` with the current version of `ember-cli-fastboot` and everything works fine. That warning is definitely [coming from ember-cli-staticboot](https://github.com/robwebdev/ember-cli-staticboot/blob/cc172859f16e56d8893b5f09d1ff952e646bd57e/addon/components/link-to.js#L10-L13).

Closing for now. Let me know if anything!
